examples/ts/tss-smc/README.md

```diff
@@ -14,10 +15,10 @@ $ nvm use
 $ npm install
 ```
 
-Use `npx ts-node` to run the desired example:
+Use `npm` to run the desired step. For example, to run step 1:
```

Comment:
This is used so that a user can run a `ts` file example directly. Maybe identify the list of scripts in their own section of the readme as well as in the example directly?